pages/public_cloud/public_cloud_network_services/getting-started-07-creating-vrack/guide.en-asia.md

```diff
@@ -97,21 +97,25 @@ For that you need to cherry-pick the correct terraform provider & resource. Find
 
 ### Step 1: Activating and managing a vRack <a name="activation"></a>
 
-Using the vRack itself is free of charge and it can be delivered within a few minutes. However, to create a vRack service, it is necessary to generate and validate an order form first.
+#### In the OVHcloud Control Panel
+
+> [!primary]
+> This does not apply to newly created projects, which are automatically delivered with a vRack. To view the vRack once the project has been created, go the `Bare Metal Cloud`{.action} menu and click on `Network`{.action} in the left tab. Click on `vRack private network`{.action} to view the vRack(s).
```

Comment:
Why don't we advise going to `Private Network` (under `Network`) inside `Public Cloud`instead?